website/static/bridging-faqs.json

```diff
@@ -1,7 +1,7 @@
+{"question": "How long does it take before I receive my funds when I initiate withdrawal from Arbitrum chains (One and Nova)?","answer": "<p>Using the official Arbitrum Bridge, the process will typically take <em>roughly</em> one week. However, some users opt to use third party fast bridges, which often bypass this delay (remember that third party bridges are created by third parties, so please DYOR!).</p>\n\n<p>There's some variability in the exact wall-clock time of the dispute window, plus there's some expected additional \"padding\" time on both ends (no more than about an hour, typically).</p>\n\n<p>The variability of the dispute window comes from the slight variance of block times. Arbitrum One's dispute window is 45818 blocks; this converts to ~1 week assuming 12 seconds per block, which is the average block time of Ethereum.</p>\n\n<p>The \"padding on both ends\" involves three events that have to occur between a client receiving their transaction receipt from a Sequencer and their L2-to-L1 message being executable. After getting their receipt,</p>\n\n<ol><li>The Sequencer posts their transaction in a batch (usually within a few minutes, though the Sequencer will wait a bit longer if the L1 is congested). Then,</li>\n<li>A validator includes their transaction in an RBlock (usually within the hour). Then, after the ~week long dispute window passes, the RBlock is confirmable, and</li>\n<li>Somebody (anybody) confirms the RBlock on L1 (usually within ~15 minutes).</li>\n</ol>\n<p>Additionally, in the rare and unlikely event of a dispute, this delay period will be extended for the dispute to resolve.</p>\n\n","key": "how-long-does-it-take-before-i-receive-my-funds-when-i-initiate-withdrawal-from-arbitrum-chains-one-and-nova"},
```

Comment:
```suggestion
{"question": "How long does it take before I receive my funds when I initiate a withdrawal from Arbitrum chains (One and Nova)?","answer": "<p>Using the official Arbitrum Bridge, the process will take <em>roughly</em> one week. However, some users opt to use third-party fast bridges, which often bypass this delay (remember that these bridges are created and maintained by third-parties, so please DYOR!).</p>\n\n<p>There's some variability in the exact wall-clock time of the dispute window, plus there's some expected additional \"padding\" time on both ends (no more than about an hour, typically).</p>\n\n<p>The variability of the dispute window comes from the slight variance of block times. Arbitrum One's dispute window is 45818 blocks; this converts to ~1 week assuming 12 seconds per block, which is the average block time of Ethereum.</p>\n\n<p>The \"padding on both ends\" involves three events that have to occur between a client receiving their transaction receipt from the Sequencer and their L2-to-L1 message being executable. After getting their receipt,</p>\n\n<ol><li>The Sequencer posts their transaction in a batch (usually within a few minutes, though the Sequencer will wait a bit longer if the L1 is congested). Then,</li>\n<li>A validator includes their transaction in an RBlock (usually within the hour). Then, after the ~week long dispute window passes, the RBlock is confirmable, and</li>\n<li>Somebody (anybody) confirms the RBlock on L1 (usually within ~15 minutes).</li>\n</ol>\n<p>Additionally, in the rare and unlikely event of a dispute, this delay period will be extended for the dispute to resolve.</p>\n\n","key": "how-long-does-it-take-before-i-receive-my-funds-when-i-initiate-withdrawal-from-arbitrum-chains-one-and-nova"},
```

Reply:
Added all suggestions except `remember that these bridges are created and maintained by third-parties`. Shouldn't in this case be "third parties"?

Reply:
I don't recall which one we agreed on to use in our docs, you can safely ignore my comment if it was `third parties` we agreed upon